### ISSUE TITLE
v1.4.1 broke GeneralImprovements's quota rollover

### ISSUE THREAD

**Issue by ProjectBots** [ZehsTeam/Lethal-Company-SellMyScrap#5]
v1.4.1 introduced a change that broke the quota rollover option of GeneralImprovements.

If the quota is reached, SellMyScrap will force it to be finished which resets the profit.

```
[Debug  :GeneralImprovements] Updated time display.
[Info   :GeneralImprovements] Updated days since death display.
[Info   :GeneralImprovements] Updated profit quota and deadline monitors
[Info   :com.github.zehsteam.SellMyScrap] 

GetOvertimeBonus();
daysUntilDeadline: 5
overtimeBonus: 110

[Info   :com.github.zehsteam.SellMyScrap] 

SyncNewProfitQuotaClientRpc();
daysUntilDeadline: 5
overtimeBonus: 110

[Info   :com.github.zehsteam.SellMyScrap] 

SyncNewProfitQuotaClientRpc();
daysUntilDeadline: 5
overtimeBonus: 110

[Debug  :me.loaforc.soundapi] Audios:UIAudio:ReachedQuotaSFX == *:UIAudio:ReachedQuotaSFX?
[Info   :GeneralImprovements] Applying surplus quota to fulfilled: $0
[Info   :GeneralImprovements] Applying surplus quota to fulfilled: $0
```

**Comment by ZehsTeam** [comment]
@ProjectBots This issue should be fixed in update [1.5.0+](https://thunderstore.io/c/lethal-company/p/Zehs/SellMyScrap/), [f99b9f3](https://github.com/ZehsTeam/Lethal-Company-SellMyScrap/commit/f99b9f3ed5888438d4a99c029b1bdc0f63a2cab4)
Set the `overrideSetNewProfitQuota` config setting to `false` to apply the fix.

**Comment by ZehsTeam** [comment]
This has now been fixed by default in the latest release v1.5.13+, [82f8e31](https://github.com/ZehsTeam/Lethal-Company-SellMyScrap/commit/82f8e31591c48b35c6759666b8f97f38a83fd7f8)
